crates/asm/txs/bridge-v1/src/commit/validation.rs

```diff
@@ -0,0 +1,167 @@
+use bitcoin::{ScriptBuf, Transaction, XOnlyPublicKey};
+use strata_primitives::l1::BitcoinXOnlyPublicKey;
+
+use crate::errors::CommitInputError;
+
+/// The index of the Claim output that must be spent by the Commit transaction.
+/// This is the first output (index 0) of the Claim transaction.
+pub const CLAIM_OUTPUT_INDEX: usize = 0;
+
+/// The index of the N/N continuation output in the Commit transaction.
+/// This output must be locked to the N/N aggregated operator key.
+pub const COMMIT_NN_OUTPUT_INDEX: usize = 1;
+
+// TODO:PG: add validatation that this transaction was created by spending N/N UTXO.
```

Comment:
This is a crucial check. I'd consider this a `FIXME`.

```suggestion
// FIXME: (@prajwolrg) add validation that this transaction was created by spending N/N UTXO.
```